manuscript/03.results.md

```diff
@@ -4,7 +4,7 @@
 
 <!-- ASM proofed. -->
 
-To illustrate the unique value of lineage measurements in analyzing intra-tumoral and drug response heterogeneity, we plotted two sets of sample lineages from measurements of the breast cancer cell line AU565 (Fig. @fig:lineage a-b). The single cell lineages reveal a striking difference in cell cycle phase durations and cell division dynamics between the two samples. Population-level measurements, on the other hand, would be unable to identify this difference as the starting and ending cell numbers are not different. Measurements that record the history of cells (e.g., CFSE staining, Luria-Delbruck experiment) can help to identify these variations within cell populations but must make assumptions about the dynamics of heterogeneity [@DOI:10.1073/pnas.1715639115; @PMID:28607484]. Lineage measurements, by contrast, provide sufficiently rich temporal information to quantify the specific structure of the phenotypic heterogeneity. 
```

Comment:
What was the comment this aims to address? It's probably good in its present form—just want to check.

Reply:
"A 1-2 sentence description of the dataset at the start of this paragraph would be helpful. and perhaps include a sentence on cell cycle phases. Additionally, this assumes the reader knows what a cell lineage is, which might not be true for a comp reader"